Fixtures/ClangModules/ModuleMapGenerationCases/Package.swift

```diff
@@ -0,0 +1,7 @@
+import PackageDescription
+
+let package = Package(
+    name: "ModuleMapGenerationCases",
+    targets: [
+		Target(name: "Baz", dependencies: ["Foo", "Bar", "Jaz"])]
```

Comment:
I would recommend renaming the subdirs here to match what the intent they are trying to test is. Otherwise, when someone comes back to this test they won't necessarily know why each case exists.